submissions/develabr/oop/index.js

```diff
@@ -0,0 +1,76 @@
+class Creature {
+  constructor(species, name, gender, saying, friends) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+    this.friends = friends;
+    this.args = ['species', 'name', 'gender', 'saying', 'friends'];
+  }
+  personInfoItems() {
+    return this.args.map((key) => this[key]).join('; ');
+  }
+}
+
+class Animal extends Creature {
+  constructor(species, name, gender, saying, friends) {
+    super(species, name, gender, saying, friends);
+    this.leg = 4;

```

Comment:
What makes `leg` so different from `name` or gender to handle `leg` property in a different way?